feature-detects/rel/prefetch.js

```diff
@@ -0,0 +1,27 @@
+/*!
```

Comment:
creating a REl directory for this test only seems coutnerintuitive. isnt there a better palce for this?

Reply:
I can move it, just was thinking a follow up pr would be for `rel=preload`

Reply:
Then I propose putting it in a "link" subdirectory, since it is a test on the link element ...

Reply:
done!